contents/tutorials/next-steps-after-installing.md

```diff
@@ -0,0 +1,160 @@
+
+Be sure to add properties to your events to better capture the variables related to the events such as the value, type, version, or something else. For example, you might want to know what version of the product someone is using or if they are a customer or not. Properties can help you here, and you can set them for users as well. 
+
+Finally, you may also want to connect to external services like [Stripe](/tutorials/stripe-payment-data) or [HubSpot](/apps/hubspot-connector) to get data. You can do that using [apps](/apps). To install an app, search for it in “browse apps,” then configure and activate it.
+
+## Building a dashboard
+
+Once you’ve got events flowing in, you’ll want to use that data to better understand your product. Instead of creating insights one at a time, you should think about creating a dashboard of multiple insights. Keep in mind what data you are capturing when you do this.
+
+Some recommendations include:
+
+- Key product analytics: usage trends, feature conversion, daily/weekly active users, retention
+- Website traffic: overall traffic, top-performing pages, traffic breakdowns, sources
+- AARRR: signup trends, usage trends, customer retention, revenue growth trends, attribution table
+- Company Scorecard: key metric trends, signups, success rate trends, conversion funnels, revenue
+
+> For both product analytics and website traffic, you can use templates to shortcut their creation. When creating a dashboard, select either option from the “template” dropdown.
+
+![Dashboard](../images/tutorials/next-steps-after-installing/dashboard.png)
+
+Creating a dashboard will give you an understanding of the different types of insights you can create, from trends to retention. It will also help you refine the events you capture to better match what you want in your dashboard.
+
+## Collaborating with your team
+
+Now that you’ve done the work on getting PostHog set up with data and dashboards, your teammates can benefit from all your hard work. You can invite teammates easily by going to your instance homepage and clicking the invite button and adding their email. This will add them to your organization with access to all your projects.
+
+![Invite](../images/tutorials/next-steps-after-installing/invite.png)
+
+To make sure they don’t mess up all your hard work, you can change their access levels in your organization settings. Setting them to “member” will prevent them from deleting or changing the settings in your project or instances. You can also set up projects you don’t want others to access as private in the project settings. 
+
+When working with others, you likely will want to [filter them out](/tutorials/filter-internal-users) of the events. To do this, go to ”project settings”, scroll down to “filter out internal and test users”  and add filters such as host does not include local development addresses and email does not include your company’s domain.
```

Comment:
Probably want to format this differently. Or just leave it as pointing to the other guide.

Reply:
Rewrote and added an image.